examples/favorites_advanced/favorites_advanced_base/lib/src/common_ui_components/puppy_card.dart

```diff
@@ -33,53 +33,57 @@ class PuppyCard extends StatelessWidget {
       : VisibilityDetector(
           onVisibilityChanged: (info) {
             if (info.visibleFraction > 0.7) {
+              print('info.visibleFraction > 0.7');
               _onVisible!(_puppy);
             }
           },
           key: Key("VisiblePuppyCard${_puppy.id}"),
           child: _buildCard(),
         );
 
-  Widget _buildCard() => GestureDetector(
-        onTap: () => _onCardPressed(_puppy),
-        child: Padding(
-          key: Key("PuppyCard${_puppy.id}"),
-          padding: EdgeInsets.symmetric(vertical: 4, horizontal: 4),
-          child: Card(
-            elevation: 2,
-            child: ListTile(
-              contentPadding: EdgeInsets.only(
-                left: 12,
-                right: 0,
-                top: 8,
-                bottom: 4,
-              ),
-              leading: Hero(
-                tag: '$PuppyCardAnimationTag ${_puppy.id}',
-                child: PuppyAvatar(
-                  asset: _puppy.asset,
-                  radius: 56,
-                ),
-              ),
-              title: SkeletonText(
-                text: _puppy.displayName,
-                height: 19,
-              ),
-              subtitle: Padding(
-                padding: EdgeInsets.only(top: 12),
-                child: SkeletonText(
-                  text: _puppy.displayCharacteristics,
-                  height: 67,
-                ),
+  Widget _buildCard() {
+    print('_buildCard');
```

Comment:
@StanislavPrime, please don't commit/push pint calls :)

Reply:
The printing in the console was removed